repository-management.md

```diff
@@ -0,0 +1,119 @@
+
+### Required Labels
+
+Some labels have a special meaning and/or might be used for automated workflows down the road. These are required across all official packages.
+
+#### Bug
+
+The `bug` label denotes an issue that is a confirmed bug where the code does not produce the expected result.
+
+#### Good First Issue
+
+The `good-first-issue` label denotes an issue that is a good entry point for new contributors wanting to get their feet wet.
+
+Good first issues are small in scope and don't require extensive technical expertise or historical project knowledge to get started.
+
+## Commits
+
+No direct commits are to be done against the `master` branch of the packages. All code changes need to go through a pull request workflow. 
+
+## Pull Requests
+
+All code changes go through a pull request.
+
+Every submitted pull request needs to go through a code review, and needs to be approved by at least one of the [committers](https://github.com/orgs/wp-cli/teams/committers).
+
+Non-trivial pull requests should be preceded by a related issue that defines the problem to solve and allows for discussion of the most appropriate solution before actually writing code.
+  
+If a pull request was submitted by one of the [committers](https://github.com/orgs/wp-cli/teams/committers), the submitter should set the "Reviewers" for that pull request to `wp-cli/committers` if a general code review is needed, or to one or more specific committer profiles if the expertise of a specific person is needed/wanted.
+
+If a pull request was submitted by an external contributor, the [committers](https://github.com/orgs/wp-cli/teams/committers) should be responsive and provide quick feedback to encourage further contributions.
+
+Apart from needing to be approved, pull requests also need to have their tests in a passing state before they can be merged. When both of these conditions are true, any [committer](https://github.com/orgs/wp-cli/teams/committers) can merge the pull request by:
+1. Ensuring that all applicable labels have been set.
+2. Ensuring that the correct milestone has been set.
+3. Ensuring that the branch is deleted after the merge (if applicable).
+4. Adapting the title of the pull-request as needed. The ideal pull request title is one that we can copy and paste into release notes as is.
+
+## Releases
+
+### WP-CLI Releases
+
+WP-CLI releases are scheduled and communicated well in advance by the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+When the release is ready, it is being prepared and executed according to the [release checklist](release-checklist.md) by one of the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+### Package Releases
+
+Package releases happen on an as-needed basis. A package is tagged for a release when:
```

Comment:
Let's keep package releases to the domain of the maintainers for now. Doing so will let us do a final review of all code in the release before it gets tagged.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/01a7f496dc2d79a5a6cef378baf276262b09ce71